Cargo.toml

```diff
@@ -1,10 +1,11 @@
 [package]
 name = "schedule"
-version = "0.1.0"
+version = "0.1.1"
```

Comment:
Don't increase the version, please. I like to use https://github.com/sunng87/cargo-release to release and that bumps the version automatically.